spartan/aztec-network/values/release-devnet.yaml

```diff
@@ -0,0 +1,34 @@
+##########
+# BEWARE #
+##########
+# You need to deploy the metrics helm chart before using this values file.
+# head to spartan/metrics and run `./install.sh`
+# (then `./forward.sh` if you want to see it)
+telemetry:
+  enabled: true
+  otelCollectorEndpoint: http://34.150.160.154:4318
```

Comment:
Should this be specifying an endpoint here, when all the others have been removed? And should it not be DNs instead of IP?

Reply:
I'll leave it in for now